### ISSUE TITLE
Improved Documentation for PL Bolts Transforms

### ISSUE THREAD

**Issue by RylanSchaeffer** [PyTorchLightning/pytorch-lightning#10711]
Good morning! I love PL Bolts and I'm super grateful you've made these models available to the community. However, I'm a bit confused by the transforms and I'd like to request better documentation. For two quick examples, suppose you want to use SwAV. The documentation says to do the following:

```
            from pl_bolts.models.self_supervised.swav.transforms import SwAVTrainDataTransform, SwAVEvalDataTransform
            from pl_bolts.transforms.dataset_normalizations import stl10_normalization
            train_transform = SwAVTrainDataTransform(
                normalize=stl10_normalization())
            eval_transform = SwAVEvalDataTransform(
                normalize=stl10_normalization())
```

But now several questions arise. The documentation suggests SwAV was pretrained on ImageNet, but here, we're using STL-10 normalization - why is that? Are these transforms appropriate only for ImageNet and STL-10? If so, what should I use for COCO or another dataset? What happens if I don't include the STL-10 normalization?

If someone who knows the library better could improve the documentation, that would be massively helpful :) 

Thanks in advance!

cc @borda

**Comment by tchaton** [comment]
Hey @RylanSchaeffer,

I believe this issue should be open on the Bolts repo. Mind doing so?

**Comment by RylanSchaeffer** [comment]
Happy to. I thought Bolts was part of Lightning?
